src/main/java/gov/usds/case_issues/controllers/TagFragment.java

```diff
@@ -15,10 +15,17 @@
 
 /**
  * A custom constraint for "one word" validations, to avoid repeating (or nearly repeating)
- * the same {@link Pattern} annotation in a million places. 
+ * the same {@link Pattern} annotation in a million places.
  */
 @Constraint(validatedBy = { })
-@Pattern(regexp="\\w[-\\w]*")
+/**
+ * \\w matchs 0 or more word chars [A-Z, a-z, 0-9, _]
+ * [-\\w]* match 0 or more words seperated by [-]
+ * Examples:
+ *   Valid: "1c58a9ab-1f8c-4743-b067-f55c60c22080", "abc_XYZ_123"
+ *   Invalid: "\n\r", "\s", "?"
+ */
```

Comment:
I commented on this on the other PR, but this should either be in the javadoc two lines up, or just be a non-javadoc block comment.